package/MDAnalysis/analysis/base.py

```diff
@@ -27,16 +27,15 @@
 classes.
 
 """
-from six.moves import range, zip
-
 import inspect
 import logging
+
 import numpy as np
 import six
-
 from MDAnalysis import coordinates
 from MDAnalysis.core.groups import AtomGroup
 from MDAnalysis.lib.log import ProgressMeter, _set_verbose
+from six.moves import range, zip
```

Comment:
Please, keep compatibility imports at the top. It helps keeping track of what behaviours change in the module.